integration-tests/utils/integration-helpers.js

```diff
@@ -90,6 +105,14 @@ const addUserToBilling = _.flow(withSignedInPage, withUserToken)(async ({ page,
   console.info(`test user was added to the billing project with the role: ${!!billingUser && billingUser.role}`)
 })
 
+const removeUserFromV1Workspace = _.flow(withSignedInPage, withUserToken)(async ({ page, billingProject, v1WorkspaceName, email }) => {
+  await page.evaluate((email, billingProject, v1WorkspaceName) => {
+    return window.Ajax().Workspaces.workspace(billingProject, v1WorkspaceName).updateAcl([{ email, accessLevel: 'NO ACCESS' }])
+  }, email, billingProject, v1WorkspaceName)
+
+  console.info(`removed user from: ${v1WorkspaceName}`)
```

Comment:
is it worth logging the user's email here?

Reply:
Sure! Why not!